size_test.go

```diff
@@ -93,13 +93,15 @@ func TestFromHumanSize(t *testing.T) {
 	assertSuccessEquals(t, 32*TB, FromHumanSize, "32Tb")
 	assertSuccessEquals(t, 32*PB, FromHumanSize, "32Pb")
 
+	assertSuccessEquals(t, 32.5*KB, FromHumanSize, "32.5kB")
+	assertSuccessEquals(t, 32.5*KB, FromHumanSize, "32.5 kB")
+	assertSuccessEquals(t, 32, FromHumanSize, "32.5 B")
```

Comment:
And there's some other situations, for example, `32.5634663 kB` which is `32563.4663 B`  which is not meaningful, and it's hard to detect this format.
But I really need `FromHumanSize` to support parsing float64, because you know `HumanSize` will print `32.56 kB`, I need this `FromHumanSize` to reverse the result.

So I think if we can't stop unmeaning input, we can still give meaningful output as some programming languages do.